Pod/Tests/TestStash.swift

```diff
@@ -0,0 +1,43 @@
+import UIKit
+import XCTest
```

Comment:
All right, let's go with XCTest.
For such simple tests I see a sense to just use XCTest, but for complex things with a lot of tests when you're lost if names like `testThisWeirdoThingyInTheContextWhenItShouldWorkAsDesired` Quick winds :smile: